src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -469,4 +471,15 @@ public void changeItemStatus(String itemStatus) {
   public void changeDeclaredLostDateTime(DateTime dateTime) {
     write(representation, LoanProperties.DECLARED_LOST_DATE, dateTime);
   }
+
+  public boolean isOverdue() {
+    return isOverdue(DateTime.now(DateTimeZone.UTC));
```

Comment:
Please use the clock instead of the `DateTime.now`

Reply:
done